app/helpers/links_helper.rb

```diff
@@ -23,13 +23,27 @@ def set_tool_tip(link)
     category = link.category
     case category
       when 'Video'
-        "Video"
+        'Click here if you watched it again'
       when 'Podcast'
-        "Podcast"
-      when 'BlogPost'
-        "Blogpost"
+        'Click here if you listened it again'
       else
-        "Tutorial"
+        'Click here if you read it again'
     end
   end
+  
+  def learning_count(link)
+    link.learn_time.count
+  end
+
+  def set_tooltip_on_count(number)
+      number == 0 ? "Never learned" : "#{number} time learned"
```

Comment:
As I said it earlier when it is > 1 time should be timeS (plural) form. 

Refer http://apidock.com/rails/ActionView/Helpers/TextHelper/pluralize